platform-ajax/src/test/java/com/softicar/platform/ajax/dom/popup/AjaxDomPopupTest.java

```diff
@@ -79,17 +87,38 @@ public void testZIndexOnClick() {
 
 		// assert clicking foreground pop-up does not change z-index
 		int zIndex = getZIndex(popup2);
-		click(popup2);
+		click(getHeader(popup2));
```

Comment:
Yes it does - nothing changed in this regard.

Clicking the header was a workaround specific to this test: The test now uses a real `DomPopup` instead of a random `div`; this change affected geometric assumptions in here (considering that Selenium targets the center of an element when "clicking".
I will once again refrain from clicking the header, and instead click at `(popup-top-left) + (0, 0)` via `clickAt(node, int, int)`. This way, we will ensure that clicking the popup content does raise the popup (by default).